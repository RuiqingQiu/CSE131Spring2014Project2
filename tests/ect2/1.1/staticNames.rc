static float a;
int b;
const auto c = true;

function : void foo(){
   static float a;
   static int b;
   static bool c;
}

function : void main(){
   static auto a = 0xFF;
   static float b;
   static bool[10] c;
   
   cout << sizeof(c) << " " << sizeof(::c) << " ";
   cout << a << " " << ::a << " ";
   cout << b << " " << ::b << " ";
   cout << c[0] << " " << c[5] << " " << ::c << endl;
}
