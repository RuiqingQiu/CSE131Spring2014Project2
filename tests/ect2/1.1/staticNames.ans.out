40 4 255 0.00 0.00 0 false false true
